torchbenchmark/models/pplbench_beanmachine/metadata.yaml

```
@@ -0,0 +1,6 @@
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+optimized_for_inference: false
+train_benchmark: false
+train_deterministic: false
```

```suggestion
train_deterministic: false
not_implemented:
    - jit: true
```
Add the `not_implemented` part to disable jit test, since torchscript can't handle this model